elastic/kibana
# [ResponseOps][Alerting] Optimize the scheduling of rule actions so it can happen in bulk

Resolves [#126511](https://github.com/elastic/kibana/issues/126511)
## Summary

We wanted to leverage bulk calls to schedule rule actions.
- Updated the actions client could support enqueueing in bulk
- Update the alerting task runner so that it could call the new bulk API

### Checklist



### To verify

- Create a rule and verify that is runs as expected, you shouldn't notice any changes from the front-end
